site/snippets/templates/buy/good-cause.php

```diff
@@ -6,40 +6,48 @@
 }
 </style>
 
-<div id="good-cause">
-	<h2 class="h2 mb-6">For a good cause? <mark class="px-1 rounded">We’ve got your back.</mark></h2>
-	<div class="prose mb-6">
-		<p>We care about a better society and the future of our planet. We offer free or discounted licenses to <strong>students, selected educational projects, social and environmental organizations, charities and non-profits</strong> with insufficient funding.</p>
-	</div>
+<section id="good-cause" class="mb-24 columns" style="--columns: 2; --columns-md: 1; --column-gap: var(--spacing-24); --row-gap: var(--spacing-6)">
+	<div>
+		<h2 class="h2 mb-6">For a good cause? <mark class="px-1 rounded">We’ve got your back.</mark></h2>
 
-	<a class="btn btn--filled mb-12" href="<?= url('good-cause') ?>">
-		<?= icon('heart') ?>
-		Learn more
-	</a>
+		<div class="prose mb-6">
+			<p>We care about a better society and the future of our planet. We offer free or discounted licenses to <strong>students, selected educational projects, social and environmental organizations, charities and non-profits</strong> with insufficient funding.</p>
+		</div>
 
-	<ul class="columns causes" style="--columns: 2; --gap: var(--spacing-12);">
-		<?php foreach (collection('cases/causes')->shuffle()->limit(2) as $case): ?>
-			<li>
-				<a href="<?= $case->link()->toUrl() ?>">
-					<figure>
-						<span class="block shadow-2xl mb-3" style="--aspect-ratio: 3/4">
-							<?= img($image = $case->image(), [
-								'alt' => $image->alt()->or('Screenshot of the ' . $case->title() . ' website'),
-								'src' => [
-									'width' => 300
-								],
-								'srcset' => [
-									400,
-									800,
-								]
-							]) ?>
-						</span>
-						<figcaption class="text-sm">
-							<?= $case->title() ?>
-						</figcaption>
-					</figure>
-				</a>
-			</li>
-		<?php endforeach ?>
-	</ul>
-</div>
+		<?php snippet('templates/buy/good-causes', [
+			'headline' => 'Free licenses',
+			'causes'   => page('buy/good-cause')->children()->filterBy('type', 'free')
+		]) ?>
+		<?php snippet('templates/buy/good-causes', [
+			'headline' => 'Discounted licenses',
+			'causes'   => page('buy/good-cause')->children()->filterBy('type', 'discounted')
+		]) ?>
+	</div>
+	<div>
+		<ul class="columns causes" style="--columns: 3; --gap: var(--spacing-12);">
+			<?php foreach (collection('cases/causes')->shuffle()->limit(6) as $case): ?>
+				<li>
+					<a href="<?= $case->link()->toUrl() ?>">
+						<figure>
+							<span class="block shadow-2xl mb-3" style="--aspect-ratio: 3/4">
+								<?= img($image = $case->image(), [
+									'alt' => $image->alt()->or('Screenshot of the ' . $case->title() . ' website'),
+									'src' => [
+										'width' => 300
+									],
+									'srcset' => [
+										400,
+										800,
+									]
```

Comment:
I realized these probably need to be updated now that we are having more columns.